### PR TITLE
Python script optimizations

### PR DESCRIPTION
**Pull request by pmcnr**
Some code optimizations for the Python script. :smiley:

### CODE REVIEW

File: __bass.py [hunk 1]
```diff
@@ -10,24 +10,24 @@
...
+    output = subprocess.check_output(args, universal_newlines=True)
+    old_env = output.splitlines()
 
     command = '{}; echo "{}"; env'.format(' '.join(sys.argv[1:]), divider)
-    output = subprocess.check_output(['bash', '-c', command])
-    if bytes != str and isinstance(output, bytes):
-        # on python 3
-        output = str(output, 'utf-8')
+    args = [BASH, '-c', command]
+    output = subprocess.check_output(args, universal_newlines=True)
```
Comment (edc) [inline __bass.py]: `output` will be of `bytes` type on Python 3. That would break the next line IIUC. Could you add back line 28-30?

Comment (pmcnr) [inline __bass.py]: Hi Eddie,
Thank you very much for your quick feedback. Adding the `universal_newlines=True` parameter to `subprocess.check_output` automatically decodes the result to a `str` as you can see below, so it should make the conversion to string you had redundant, no?
```python
Python 3.4.3 (default, Mar 25 2015, 17:13:50) 
[GCC 4.9.2 20150304 (prerelease)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> type(subprocess.check_output('ls', universal_newlines=True))
<class 'str'>
```

Comment (edc) [inline __bass.py]: TIL. This is great!

**Comment by pmcnr** [comment]
:smiley: Thanks!

**Pull request merged**